content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+        volumeMounts:
+        - name: cribl-stream-config
+          mountPath: /etc/cribl-stream
+      volumes:
+      - name: cribl-stream-config
+        configMap:
+          name: cribl-stream-config
+```
+
+#### Deploying Cribl Stream using Helm Charts
+
+The Cribl Stream helm charts can be found on github (<https://github.com/criblio/helm-charts>). This assumes that the namespace is set to `cribl`. 
+
+﻿Log into cloud CLI or jump box and issue the following commands:
+
+```shell
+export KUBECONFIG=<path_to_kube_settings>
+kubectl get nodes -n cribl
+kubectl get svc  -n cribl
+```
+
+Label the leader node and the worker nodes:
+
+```shell
+kubectl label nodes <leader_node> stream=leader
+kubectl label nodes <worker_node> stream=worker
+```
+
+Validate by running:
+
+```shell
+kubectl get nodes --show-labels
+```
+
+Create and modify the `values.yaml` file for workers and leader nodes. For the leader nodes, create a file named `Leader_values.yaml` and modify line 97:
+
+```yaml
+nodeSelector:
+     stream: leader
+```
+
+For the worker nodes, create a file named `Worker_values.yaml` and modify line 97:
+
+```yaml
+nodeSelector:
+     stream: worker
+```
+
+Next set the labels for your workers and leader node.
+
+First get a list of all the nodes and see the labels associated with them:
+
+```shell
+kubectl get nodes --show-labels
+```
+
+Now, identify the nodes and make sure to label the nodes according to their role for this deployment. 
+
+Here is an example of setting the host `k8s-cribl-master-t497j-92m66.gl-hpe.net` as a leader:
+
+```shell
+kubectl label nodes k8s-cribl-master-t497j-92m66.gl-hpe.net stream=leader
+```
+
+Here is an example of setting the host `k8s-cribl-wor8v32g-cdjdc-8tkhn.gl-hpe.net` as a worker node:
+
+```shell
+kubectl label nodes k8s-cribl-wor8v32g-cdjdc-8tkhn.gl-hpe.net stream=worker
+```
+
+If you accidentally label a node and want to remove or overwrite the label, you can use this command:
+
+```shell
+kubectl label nodes k8s-cribl-wor8v32g-cdjdc-876nq.gl-hpe.net stream=worker --overwrite=true
+```
+
+Once the labels have been set, now you are ready to run the helm command and deploy Cribl Stream on your environment. The first command will deploy the Cribl Leader node:
+
+```shell
+helm install --generate-name cribl/logstream-leader -f leader_values.yaml -n cribl
+```
+
+When successful, you will see a similar output like the one below: 
+
+```shell
+NAME: logstream-leader-1696441333
+LAST DEPLOYED: Wed Oct  4 17:42:16 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Note that this will deploy the leader node with the parameters found in the `leader_values.yaml` file and into the namespace `cribl`. 
+
+Next, deploy the worker nodes using the `worker_values.yaml` file into the namespace `cribl`.
+
+```shell
+helm install --generate-name cribl/logstream-workergroup -f workers_values.yaml
+```
+
+When successful, you will see a similar output like the one below:
```

Comment:
Rephrase to:
"When successful, you will see output similar to what's shown below:"